### PR TITLE
z, zip, zless, zola, zopflipng, zoxide: add French translation

### PR DESCRIPTION
**Pull request by sebastientinel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.fr/common/z.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# z
+
+> Recherche les répertoires les plus utilisés et permet une navigation rapide à l'aide de chaine de caractères ou de regex.
+> Plus d'informations: <https://github.com/rupa/z>.
+
+- Aller dans un répertoire qui contient "foo" dans son nom:
```
Comment (navarroaxel) [inline pages.fr/common/z.md:6]: I remember that the French language use a space before `:`. Please fix all the example descriptions.
```suggestion
- Aller dans un répertoire qui contient "foo" dans son nom :
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @sebastientinel, welcome and thanks for your submission. I left 1 comment.

**Comment by sebastientinel** [comment]
@navarroaxel Thank you for the feedback. 
Fixed it.

Should be good to go!

**Review by navarroaxel — APPROVED** [review]

File: pages.fr/common/zoxide.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+> Garde une trace des répertoires les plus utilisés.
+> Utilise un algorithme de classement pour identifier le meilleur résultat.
+> Plus d'informations : <https://github.com/ajeetdsouza/zoxide>.
+
+- Aller au répertoire avec le meilleur classement qui contient "foo" dans son nom :
+
+`zoxide query {{foo}}`
+
+- Aller au répertoire avec le meilleur classement qui contient "foo" et "bar" dans son nom ::
```
Comment (navarroaxel) [inline pages.fr/common/zoxide.md:11]:
```suggestion
- Aller au répertoire avec le meilleur classement qui contient "foo" et "bar" dans son nom :
```

File: pages.fr/common/zoxide.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`zoxide query {{foo}} {{bar}}`
+
+- Démarre une recherche de répertoire interactive (requires `fzf`) :
+
+`zoxide query --interactive`
+
+- Ajoute un répertoire ou incrémente son classement :
+
+`zoxide add {{chemin/du/répertoire}}`
```
Comment (navarroaxel) [inline pages.fr/common/zoxide.md:21]: Could you please fix the `path/to/dir` to `path/to/directory` in https://github.com/tldr-pages/tldr/blob/master/pages/common/zoxide.md?

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @sebastientinel, I left some comments.

File: pages.fr/common/zoxide.md (hunk 3 above)
Comment (sebastientinel) [reply pages.fr/common/zoxide.md:21]: @navarroaxel Sure, do you want me to do the same with https://github.com/tldr-pages/tldr/blob/master/pages/common/zip.md?

Comment (navarroaxel) [reply pages.fr/common/zoxide.md:21]: Oh, I didn't catch that. Yep, but in a new PR.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Pull request merged**